### PR TITLE
Fix bug in sktwocnt for the global hybrid B3LYP

### PR DESCRIPTION
**Pull request by vanderhe**
Bug fix for the fraction $\alpha$ of Fock-type exchange not being correctly propagated to the two-center integration routines, when using the global hybrid functional B3LYP. This unfortunately necessitates to recalculate the associated regression tests and to create a bug fix release as soon as possible.

### CODE REVIEW

File: sktwocnt/lib/twocnt.f90 [hunk 1]
```diff
@@ -253,7 +253,7 @@ subroutine get_twocenter_integrals(inp, imap, skham, skover)
       call xc_f03_func_init(xcfunc_c, XC_GGA_C_PBE, XC_UNPOLARIZED)
     elseif (inp%iXC == 7) then
```
Comment (bhourahine) [inline sktwocnt/lib/twocnt.f90:254]: Any reason these are suddenly numerical values instead of from an enum() ?

File: sktwocnt/lib/twocnt.f90 [hunk 2]
```diff
@@ -253,7 +253,7 @@ subroutine get_twocenter_integrals(inp, imap, skham, skover)
...
       call xc_f03_func_init(xcfunc_xc, XC_HYB_GGA_XC_B3LYP, XC_UNPOLARIZED)
-      call xc_f03_func_set_ext_params(xcfunc_xc, [0.20_dp, 0.72_dp, 0.81_dp])
+      call xc_f03_func_set_ext_params(xcfunc_xc, [inp%camAlpha, 0.72_dp, 0.81_dp])
```
Comment (bhourahine) [inline sktwocnt/lib/twocnt.f90:256]: Probably needs a comment, as there are still hard coded values on the line.

File: sktwocnt/prog/input.f90 [hunk 3]
```diff
@@ -113,10 +113,14 @@ subroutine readInput(inp, fname)
...
 
     if (inp%iXC == xcFunctional%HYB_B3LYP) then
+      ! 20% HFX hard-coded at the moment
+      inp%camAlpha = 0.2_dp
+      inp%camBeta = 0.0_dp
       call nextline_(fp, iLine, line)
       read(line, *, iostat=iErr) inp%nRadial, inp%nAngular, inp%ll_max, inp%rm
       call checkerror_(fname, line, iLine, iErr)
     elseif (inp%iXC == xcFunctional%HYB_PBE0) then
+      inp%camBeta = 0.0_dp
```
Comment (bhourahine) [inline sktwocnt/prog/input.f90:123]: Is Alpha never referenced on this path (the variable looks to be undefined)?

File: sktwocnt/lib/twocnt.f90 (hunk 1 above)
Comment (vanderhe) [reply sktwocnt/lib/twocnt.f90:254]: No good reason, I just overlooked those occurrences when cleaning up, it is on my to-do list.

File: sktwocnt/lib/twocnt.f90 (hunk 2 above)
Comment (bhourahine) [reply sktwocnt/lib/twocnt.f90:256]: How about something like: Alpha is adjustable for B3LYP, but X and Y are from ...

Comment (vanderhe) [reply sktwocnt/lib/twocnt.f90:256]: It is not yet adjustable from the user's perspective of course, but yes for this routine it effectively is. The other values are just the standard parametrization of B3LYP, I can add a reference to the paper.

**Review by bhourahine — APPROVED** [review]
LGTM, but obviously needs a release discussion at the next dev meeting.

**Pull request merged**